packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -163,13 +168,122 @@ class IosDomainError extends DomainError {
+        'substitution variable format. Ref - '
+        'https://developer.apple.com/documentation/bundleresources/applinks/substitutionvariables',
+  );
+  static const defaultsFormat = AASAfileFormatSubCheck(
+    'Applinks defaults format',
+    'This test checks for `applinks.defaults`. Ref - '
+        'https://developer.apple.com/documentation/bundleresources/applinks/defaults'
+        'This checks if the `Defaults` property only holds `caseSensitive` and'
+        '`percentEncoded`.',
+  );
+
+  static const defaultsPercentEncodedFormat = AASAfileFormatSubCheck(
+    'Applinks defaults percent encoded format',
+    'This test checks if the `applinks.defaults.percentEncoded` property only holds a boolean.',
+  );
+
+  static const defaultsCaseSensitiveFormat = AASAfileFormatSubCheck(
+    'Applinks defaults case sensitive format',
+    'This test checks if the `applinks.defaults.caseSensitive` property only holds a boolean.',
+  );
+
+  static const detailsFormat = AASAfileFormatSubCheck(
+    'Applinks details format',
+    'This test checks for `applinks.details`. Ref - '
+        'https://developer.apple.com/documentation/bundleresources/applinks/details',
+  );
+
+  static const detailsAppIdFormat = AASAfileFormatSubCheck(
+    'Applinks details appIDs format',
+    'This test checks if the `applinks.details.appID` property is an array of Strings.',
+  );
+
+  static const detailsPathsFormat = AASAfileFormatSubCheck(
+    'Applinks details paths format',
+    'This test checks if the `applinks.details.paths` property is an array of Strings.',
+  );
+
+  static const detailsDefaultsFormat = AASAfileFormatSubCheck(
+    'Applinks details default format',
+    'This test checks for `applinks.details.default`. Ref - '
+        'https://developer.apple.com/documentation/bundleresources/applinks/details/default'
+        'This checks if the `Defaults` property only holds `caseSensitive` and'
+        '`percentEncoded`.',
+  );
+
+  static const detailsDefaultsPercentEncodedFormat = AASAfileFormatSubCheck(
+    'Applinks defaults percent encoded format',
+    'This test checks if the `applinks.details.defaults.percentEncoded` property only holds a boolean.',
+  );
+
+  static const detailsDefaultsCaseSensitiveFormat = AASAfileFormatSubCheck(
+    'Applinks defaults case sensitive format',
+    'This test checks if the `applinks.details.defaults.caseSensitive` property only holds a boolean.',
+  );
+
+  static const componentFormat = AASAfileFormatSubCheck(
+    'Applinks details components format',
+    'This test checks for `applinks.details.components` below. Ref - '
+        'https://developer.apple.com/documentation/bundleresources/applinks/details/components',
+  );
+
+  static const componentPathFormat = AASAfileFormatSubCheck(
+    'Applinks details components path format',
+    'This test checks if the `applinks.details.components.path` property only holds a string.',
```

Comment:
nit: we should be consistent with whether we capitalize "s" in the word string throughout these errors

Reply:
good eyes!